File: Android/app/src/main/java/pl/me/inwentaryzacja/webapiDA/models/Product.java
package pl.me.inwentaryzacja.webapiDA.models;

public class Product {
    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    private String code;
    private String description;
}

File: WebApi/Model/Models/Product.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Model.Models
{
    [Serializable]
    public class Product
    {
        public Product(string code, string description)
        {
            this.code = code;
            this.description = description;
        }

        private string code;
        private string description;

        public string Code
        {
            get { return code; }
            set { code = value; }
        }

        public string Description
        {
            get { return description; }
            set { description = value; }
        }

    }
}